bam_sort.c

```diff
@@ -1227,6 +1229,12 @@ int bam_merge_core2(int by_qname, const char *out, const char *mode,
         if ((translation_tbl+i)->lost_coord_sort && !by_qname) {
             fprintf(stderr, "[bam_merge_core] Order of targets in file %s caused coordinate sort to be lost\n", fn[i]);
         }
+
+        if (!refs && cram_get_refs(fp[i]))
+            refs = cram_get_refs(fp[i]);
+
+        if (refs && hts_set_opt(fp[i], CRAM_OPT_SHARED_REF, refs))
+            return -1;  // FIXME: memory leak
```

Comment:
I was just copying the coding style at the time (there are LOTS of that same comment in my version, ugh).

Will merge with upstream and fix this additional one (+makefile).